lms/templates/emails/activation_email.txt

```diff
@@ -12,29 +14,30 @@ ${_("To get started, please activate your account by clicking on the link below"
   http://${ site }/activate/${ key }
 % endif
 
-% if settings.PLATFORM_NAME == "edX":
-${_("Activation ensures that you can register for {platform_name} courses and"
-      " access the courseware."
-      " If you require assistance, please use our web form at"
-      " {contact_us} or email {info_address}.").format(
-             platform_name=settings.PLATFORM_NAME,
-             contact_us='https://www.edx.org/contact-us',
-             info_address=settings.CONTACT_EMAIL
-      )}
+% if settings.FEATURES.get('IS_EDX_DOMAIN'):
+${_("Activating your account ensures you can sign up for and "
+    "take any of the hundreds of {platform_name} courses."
+  ).format(platform_name=settings.PLATFORM_NAME)}
+
+${_("If you need help please use our web form at "
+    "{contact_us_url} or email {info_email_address}."
+  ).format(
+    contact_us_url="https://www.edx.org/contact-us",
+    info_email_address=settings.CONTACT_EMAIL
+  )}
 
 ${_("We hope you enjoy learning with {platform_name}!").format(
-        platform_name=settings.PLATFORM_NAME
+    platform_name=settings.PLATFORM_NAME
 )}
 
 ${_("The {platform_name} Team").format(platform_name=settings.PLATFORM_NAME)}
 
 ${_("This email was automatically sent by {site_name} because someone "
-      "attempted to create an {platform_name} account using this email address. If you "
-      "did not attempt to create this account and do not activate the account, "
-      "you will no longer receive emails from {platform_name}.").format(
-        platform_name=settings.PLATFORM_NAME,
-	site_name=settings.SITE_NAME
-)}
+      "attempted to create an {platform_name} account using this email address."
+    ).format(
+      platform_name=settings.PLATFORM_NAME,
+      site_name=settings.SITE_NAME
```

Comment:
Nitpick: the order of these arguments is opposite the order in which they appear in the format string.